lib/spaceship/portal/device.rb

```diff
@@ -27,12 +27,28 @@ class Device < PortalBase
       #   "c"
       attr_accessor :status
 
+      # @return (String) Model (can be nil)
+      # @example
+      #   'iPhone 6', 'iPhone 4 GSM'
+      attr_accessor :model
+
+      # @return (String) Device class
+      # @example
+      #   'pc'     - Apple TV
+      #   'watch'  - Apple Watch
+      #   'ipad'   - iPad
+      #   'iphone' - iPhone
+      #   'ipod'   - iPod
+      attr_accessor :device_class
```

Comment:
Could we rename this to `device_type`, as it's not a Ruby class and it might be confusing for library users.